### DISCUSSION TITLE
Just switched to Jyers Marlin 1.3.3 - 1.3.4 from the Smith 3D roll and havin some issues

### DISCUSSION THREAD

**Discussion by phodara** [Jyers/Marlin#1022] [Q&A] [answered]
I have to say I really like the interface and features of this version, but I am having two serious issues.
First: I am running an Ender 3V2 (v4.2.2MB), Octopi on a RPi 3, and A BLTouch, currently at JeyrsMarlin 1.3.4
1. The notorious M112 probe error. I went ahead and set 'error handling ->ignore warning' and that seemed to quiet that issue. But I think that's a kludge and would feel more comfortable knowing what the real issue is and having it fixed. I don't think its a hardware issue I never had it with other versions of marlin.

2. My TP-Link end of job power off no longer works.  For me this is a pretty major issue. I use a TP-Link power switch for the printer and a plugin in Octopi to switch off ac power when the print job finishes.  The weird thing is the Plugin's GUI button allows me to turn the printer on and off from octopi but not automatically at the end of job.  I went ahead and completely wiped octopi and re-flashed and reinstalled only this plugin.  There are two tp-link plugins and I tried both and neither work.

I am not sure its related but when a print job completes there is an annoying 'confirm' button on the printers LCD display.  I am not sure what the point of this button is which you have to click through (I couldn't find docs maybe my ignorance).  I am wondering if this preventing the job to complete for octopi to send the shutdown command to the printer.
ANy assistance would be greatly appreciated.

**Comment by kted** [comment]
I use the Tasmota plugin with a Sonoff, and it works just fine. The printer switches off after a predetermined time, even if there is no one around to acknowledge the "Confirm" message.

**Comment by Jyers** [comment, marked as answer]
As far as the M112 error. It's a Marlin/Octoprint bug. For whatever reason sometimes OctoPrint thinks an error of some kind has occurred and it throws an M112 error to try to halt the printer. The reason you haven't seen it in other marlin versions is because other V2 versions don't actually tell you when an error occurs, that was something I added to mine to make sure the user was aware of what happened. I assume the M112 isn't actually able to halt the printer which is why other versions don't see the issue. I could silence it on the fimware side, but that would be just the same as doing it in octoprint.

As far as the plugin, it's most likely not a problem with the firmware. The confirm screen is merely just a nice end screen for you to see how long your print took or take a picture or something. It has absolutely no impact on the function of printing. If you start a print while it's still on the confirm screen everything will still work as normal.

**Comment by phodara** [comment]
Thank you for the followup. Everything is working now. Great firmware build
.  I think my issues stemmed from corrupted GCode from Cura.  I recreated
my profiles and that seemed to fix the problem.

On Sat, Jun 19, 2021 at 8:26 PM Jyers ***@***.***> wrote:

> As far as the M112 error. It's a Marlin/Octoprint bug. For whatever reason
> sometimes OctoPrint thinks an error of some kind has occurred and it throws
> an M112 error to try to halt the printer. The reason you haven't seen it in
> other marlin versions is because other V2 versions don't actually tell you
> when an error occurs, that was something I added to mine to make sure the
> user was aware of what happened. I assume the M112 isn't actually able to
> halt the printer which is why other versions don't see the issue. I could
> silence it on the fimware side, but that would be just the same as doing it
> in octoprint.
>
> As far as the plugin, it's most likely not a problem with the firmware.
> The confirm screen is merely just a nice end screen for you to see how long
> your print took or take a picture or something. It has absolutely no impact
> on the function of printing. If you start a print while it's still on the
> confirm screen everything will still work as normal.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jyers/Marlin/discussions/1022#discussioncomment-893341>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEJTDDMACXMWP2T27GNQX4DTTUYT7ANCNFSM463VE4ZQ>
> .
>


-- 
Paul Hodara
